components/PromiseCard.tsx

```diff
@@ -39,8 +39,8 @@ export default function PromiseCard({
   const impactRationale =
     promise.bc_promise_rank_rationale || "No rationale provided.";
   let impactIcon = null;
-  let impactRankStr = String(impactRankRaw).toLowerCase();
-  let impactRankNum = Number(impactRankRaw);
+  const impactRankStr = String(impactRankRaw).toLowerCase();
```

Comment:
`Warning: 'titleClasses' is never reassigned. Use 'const' instead.  prefer-const`